database/migrations/2015_01_01_100000_create_country_table.php

```diff
@@ -8,7 +8,7 @@ class CreateCountryTable extends Migration
 {
     public function up()
     {
-        Schema::create('country', function (Blueprint $table) {
+        Schema::create('countries', function (Blueprint $table) {
```

Comment:
I'm assuming this was a bug as well - that the table name should be plural like other tables.

Reply:
Nice catch. Thanks!